### PR TITLE
New command to launch service logs

### PR DESCRIPTION
**Pull request by mburke5678**
https://trello.com/c/uwFE9Bh9/938-new-commands-for-logging-was-restart-services

### CODE REVIEW

**Comment by mburke5678** [comment]
@sdodson 
Is your team involved with the new logging commands and new logging methods?
If so, is there any information I can review to learn more, sample outputs, etc.?

**Comment by sdodson** [comment]
Yes, @ravisantoshgudimetla is also working changes in this area. As far as what's currently merged this PR looks fine to me.

File: day_two_guide/topics/proc_restore_etcd.adoc [hunk 1]
```diff
@@ -88,7 +88,7 @@ command adding the `--force-new-cluster` option:
 . Check for error messages:
 +
 ----
-$ journalctl -fu etcd.service
+$ master-logs api api
```
Comment (ravisantoshgudimetla) [inline day_two_guide/topics/proc_restore_etcd.adoc]: I think these would work with docker runtime only.

**Comment by mburke5678** [comment]
@ravisantoshgudimetla can you PTAL also? Is there anything we need to add in addition to changing the command? Do you have sample output for the commands that might be helpful?

**Comment by ravisantoshgudimetla** [comment]
@mburke5678 As I updated earlier, I am working on a PR for removing some of these scripts(which are docker dependent) and adding few more. https://github.com/openshift/openshift-ansible/pull/8990, so I think it is better to hold this PR till then.

**Comment by mburke5678** [comment]
@ravisantoshgudimetla My apologies. I wasn't aware that openshift/openshift-ansible#8990 also affected logging.  Thank you for the clarification.

**Comment by ravisantoshgudimetla** [comment]
@mburke5678 - Yesterday, it has been decided to drop systemd shims, so this PR can go in.

**Comment by mburke5678** [comment]
@sdodson @ravisantoshgudimetla All I have done for this PR is to swap out the old commands for the new commands. is this all we need for this feature?
Do we want to add information on what is in each log? Example output? 
There is some information on Service Logs here:
http://file.rdu.redhat.com/mburke/trello-view-log-commands/security/monitoring.html#security-monitoring-logs

What more can we add?

**Comment by mburke5678** [comment]
@geliu2016 PTAL Can you review the changes I made to the logging commands? If the team wants to add more detail, I can do a follow-up PR. Would like to close on the command changes for 3.10.

**Review by geliu2016 — COMMENTED** [review]
LGTM, thx

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL Small find/replace changes.

File: security/monitoring.adoc [hunk 2]
```diff
@@ -89,7 +89,7 @@ This section describes the types of operational logs produced on the cluster.
 [[security-monitoring-service-logs]]
 === Service Logs
 
-{product-title} produces logs with each *systemd* service that is running on a host:
+{product-title} produces logs for services that are running on static pods in a cluster:
```
Comment (kalexand-rh) [inline security/monitoring.adoc]: s/are running/run

File: security/monitoring.adoc [hunk 3]
```diff
@@ -98,14 +98,15 @@ ifdef::openshift-origin[]
...
-- atomic-openshift-master-controllers
+- API
+- Controllers
 - etcd
 - atomic-openshift-node
 endif::[]
 
-These logs are intended more for debugging purposes than for security auditing. They can be
-retrieved per host with `journalctl` or, in clusters with the aggregated logging stack deployed, in the logging _.operations_ indexes (possibly
+These logs are intended more for debugging purposes than for security auditing. The logs can be
```
Comment (kalexand-rh) [inline security/monitoring.adoc]: s/The logs can be retrieved for/You can retrieve logs for

File: security/monitoring.adoc [hunk 4]
```diff
@@ -98,14 +98,15 @@ ifdef::openshift-origin[]
...
+retrieved for each service with the `master-logs api api`, `master-logs controllers controllers`, or `master-logs etcd etcd` commands. 
+Or, you can retrieve logs in clusters with the aggregated logging stack deployed, in the logging _.operations_ indexes (for example,
```
Comment (kalexand-rh) [inline security/monitoring.adoc]: Maybe "If your cluster runs an aggregated logging stack, like a xref:../install_config/aggregate_logging.adoc#aggregated-ops[Ops cluster], cluster administrators can retrieve logs from the logging _.operations_ indexes." ?

**Review by kalexand-rh — COMMENTED** [review]
Couple of suggestions, otherwise LGTM

**Pull request merged**